dotnet/src/webdriver/EncodedFile.cs

```diff
@@ -19,41 +19,41 @@
 
 using System;
 
+#nullable enable
+
 namespace OpenQA.Selenium
 {
     /// <summary>
     /// Represents a file transmitted over the wire as a base64-encoded string.
     /// </summary>
     public abstract class EncodedFile
     {
-        private string base64Encoded = string.Empty;
-        private byte[] byteArray;
-
         /// <summary>
         /// Initializes a new instance of the <see cref="EncodedFile"/> class.
         /// </summary>
         /// <param name="base64EncodedFile">The file as a Base64-encoded string.</param>
+        /// <exception cref="ArgumentNullException">If <paramref name="base64EncodedFile"/> is <see langword="null"/>.</exception>
+        /// <exception cref="FormatException">
```

Comment:
I am not so experienced guy writing docs, but does it mean that we should mention all possible exception types, even raised by downstream flow?!

Reply:
Only because we are exposing the `EncodedFile` constructor to other users! Users can derive this type and do something like this:

```csharp
public MyFile : EncodedFile
{
    public MyFile(string encodedFile) : base(encodedFile)
    {
    }
}
```

 If this is not intended for users to do, we can change this constructor to `private protected` and we do not need to bother documenting it at all!

Reply:
I googled, seems a good library should document all known exceptions that might be raised by the member, even transitively.